src/graphx/graphx.h

```diff
@@ -756,6 +757,72 @@ void gfx_FillCircle_NoClip(uint24_t x,
 #define gfx_Circle_NoClip(x, y, radius) \
 gfx_Circle((x), (y), (radius))
 
+/**
+ * Draws an unclipped filled ellipse.
+ *
+ * This is measured from the top left origin of the screen.
+ * Performs faster than using gfx_FillEllipse, but can cause corruption if used outside the bounds of the screen.
+ * 
+ * @warning
+ * When using too large radii (> 128) the upper and lower half of the ellipse might get corrupted due to
+ * integer overflow.
+ * 
+ * @param x X coordinate of the center.
+ * @param y Y coordinate of the center.
+ * @param a The horizontal radius of the ellipse.
+ * @param b The vertical radius of the ellipse.
+ */
+void gfx_FillEllipse_NoClip(int x, int y, int a, int b);
+
+/**
+ * Draws a filled ellipse.
+ *
+ * This is measured from the top left origin of the screen.
+ * 
+ * @warning
+ * When using too large radii (> 128) the upper and lower half of the ellipse might get corrupted due to
+ * integer overflow.
+ * 
+ * @param x X coordinate of the center.
+ * @param y Y coordinate of the center.
+ * @param a The horizontal radius of the ellipse.
+ * @param b The vertical radius of the ellipse.
+ */
+void gfx_FillEllipse(int x, int y, int a, int b);
```

Comment:
Please use the following types: `int24_t x, int24_t y, uint8_t a, uint8_t b`.

The same applies to `gfx_Ellipse`.

Reply:
> Please use the following types: `int24_t x, int24_t y, uint8_t a, uint8_t b`.
> 
> Also applies to `gfx_Ellipse`.

Pretty sure the radii should be int24_t?